index.html

```diff
@@ -47,6 +48,48 @@ <h3 class="octopus__subtitle">
           </button>
         </div>
       </section>
+      <section class="diver">
```

Comment:
Where is container?

Reply:
it is not possible to put a container, because then there is no indentation between the columns.

Reply:
No, it requires a container. It has no effect on the gap between columns